modules/tensor_mechanics/doc/content/syntax/Modules/TensorMechanics/CohesiveZoneMaster/index.md

```diff
@@ -0,0 +1,36 @@
+# Cohesive Zone Model
+
+## Description
+
+The implemented cohesive zone model (CZM) is based on a Discrete Galerkin approach and therefore does not require cohesive elements. The CZM is formulated in terms of traction separation laws and requires three ingredients:
+
+1. `BreakMeshByBlockGenerator`
+2. `CZMInterfaceKernels`
+3. `CZMMatarials`
+
+The [BreakMeshByBlockGenerator](BreakMeshByBlockGenerator.md) is utilized to create the cohesive zone interface  by splitting a monolithic mesh into blocks by adding the required nodes and boundaries between each block pair.
+The split mesh allows to compute a displacement jump $\Delta u$ at each quadrature point on the interface.
+
+### Small Deformations
+
+A [CZMInterfaceKernel](CZMInterfaceKernel.md) utilizes the displacement jump $\Delta u$ to add the appropriate residual to the system and to provide the correct Jacobian. The current implementation of the `CZMInterfaceKernel` assumes the traction separation laws being only dependent from the displacement variables. Note that one `CZMInterfaceKernel` is required for each displacement component.
```

Comment:
Add to the end of line 16: 'The `CohesiveZoneMaster` block is used to simplify the process of setting up the necessary inputs required for cohesive zone modeling. It creates the appropriate number of `CZMInterfaceKernel` objects for the model dimensionality.

Delete line 17 below.